src/templates/resources/footer.json

```diff
@@ -2623,6 +2714,8 @@
                 "text": "[Ṽîéŵẋẋ åļļẋ ļöçåţîöñšẋẋẋ]"
             }
         ],
+        "brands": "Brands",
+        "brandLinks": [],
```

Comment:
Maybe have some test data here and write some new tests for the new column/layout?

Reply:
Will do

Reply:
Doesn't look like we have any existing tests over the handlebars content, so without setting up new tooling this wouldn't be possible. All of the existing tests push html strings in and test JS behaviours.

Reply:
@simonsandersje  any objections to me pulling out implementation of handle bars test into another ticket? I don't want to spend too long setting up the tooling now, given we are looking to stop using the f-footer handlebars stuff in May.